dt/uni04delta/edpm-pre-ceph/kustomization.yaml

```diff
@@ -0,0 +1,48 @@
+---
+apiVersion: kustomize.config.k8s.io/v1alpha1
+kind: Component
+
+transformers:
+  # Set namespace to OpenStack on all namespaced objects without a namespace
+  - |-
+    apiVersion: builtin
+    kind: NamespaceTransformer
+    metadata:
+      name: _ignored_
+      namespace: openstack
+    setRoleBindingSubjects: none
+    unsetOnly: true
+    fieldSpecs:
+      - path: metadata/name
+        kind: Namespace
+        create: true
+
+components:
+  - ../../../lib/dataplane
+
+patches:
+  - target:
+      kind: OpenStackDataPlaneNodeSet
+      name: .*
+    patch: |-
+      - op: replace
+        path: /metadata/name
+        value: ceph-nodeset
+  - target:
+      kind: OpenStackDataPlaneDeployment
+      name: .*
+    patch: |-
+      - op: replace
+        path: /metadata/name
+        value: edpm-deployment-pre-ceph
+      - op: replace
+        path: /spec/nodeSets
+        value: [ceph-nodeset]
+  - target:
+      kind: Secret
+      name: nova-migration-ssh-key
+    patch: |-
+      - op: add
+        path: /metadata/annotations
+        value:
+          config.kubernetes.io/local-config: true
```

Comment:
Now that you have added this, you can remove the symlinks for...

- `dataplane-ssh-secret.yaml`
- `openstackdataplanedeployment.yaml`
- `openstackdataplanenodeset.yaml`

...from this `dt/uni04delta/edpm-pre-ceph` dir.

Reply:
Done